packages/react-components/react-nav-preview/src/components/NavDrawer/useNavDrawerStyles.styles.ts

```diff
@@ -1,8 +1,9 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
-import type { NavDrawerSlots, NavDrawerState } from './NavDrawer.types';
+import type { NavDrawerState } from './NavDrawer.types';
+import { InlineDrawerSlots, useInlineDrawerStyles_unstable } from '@fluentui/react-drawer';
```

Comment:
```suggestion
import { InlineDrawerSlots, useInlineDrawerStyles_unstable } from '@fluentui/react-drawer';
import type { NavDrawerState } from './NavDrawer.types';
```

I'm also missing a type import in line 2 that should be at the bottom